src/apps/loyalty/containers/inquiries/index.tsx

```diff
@@ -40,6 +84,33 @@ class Inquiries extends React.Component<RelayProps, any> {
     })
   }
 
+  handleTextboxChange(e) {
```

Comment:
It looks like you can type the event as `React.FormEvent<HTMLTextAreaElement>`.

[`FormEvent`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ddce4ffcfeb85b00cf380d5565f7f71c0eb69640/types/react/index.d.ts#L312) is [just a `SyntheticEvent`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ddce4ffcfeb85b00cf380d5565f7f71c0eb69640/types/react/index.d.ts#L277-L294), which for [the `target` property](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ddce4ffcfeb85b00cf380d5565f7f71c0eb69640/types/react/index.d.ts#L290-L291) has [this tidbit](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/11508#issuecomment-256045682) which explains that we should use [`currentTarget`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ddce4ffcfeb85b00cf380d5565f7f71c0eb69640/types/react/index.d.ts#L279) instead and that will get cast as `HTMLTextAreaElement` through `T`.